src/zoid/buttons/component.jsx

```diff
@@ -1299,6 +1300,17 @@ export const getButtonsComponent: () => ButtonsComponent = memoize(() => {
         bodyParam: getEnv() === ENV.LOCAL || getEnv() === ENV.STAGE,
       },
 
+      sdkSource: {
+        type: "string",
+        value: () => {
```

Comment:
I think the short form would work here, since `sdkSourceValue` isn't used anywhere else (and increases memory usage for a time):
```suggestion
        value: () => getSDKIntegrationSource(),
```